node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/InactiveExpirer.java

```diff
@@ -45,8 +52,12 @@ protected void expire(List<Node> expired) {
 
     @Override
     protected boolean isExpired(Node node) {
-        return    super.isExpired(node)
-               || node.allocation().get().owner().instance().isTester();
+        return super.isExpired(node, timeout(node)) ||
```

Comment:
Need to update `Nodes::deactivate` to move `config` nodes to `inactive`?

Reply:
Not sure what you mean? According to the tests I added, this works as intended.

Reply:
Interesting, never mind then.
I thought they would be moved straight to `dirty` since they are not stateful (only content nodes are by default), but I see now we explicitly set them to stateful in `ConfigServerLikeApplication`